compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Hoogle.hs

```diff
@@ -101,20 +101,21 @@ cls2hoogle ClassDoc{..} = concat
                  ++ maybe [] ((:["=>"]) . type2hoogle) cl_super
                  ++ wrapOp (unTypename cl_name) : cl_args
       , "" ]
-    , concatMap (fct2hoogle . addToContext) cl_functions
+    , concatMap classMethod2hoogle cl_methods
+    ]
+
+classMethod2hoogle :: ClassMethodDoc -> [T.Text]
+classMethod2hoogle ClassMethodDoc{..} | cm_isDefault = [] -- hide default methods from hoogle search
+classMethod2hoogle ClassMethodDoc{..} = concat
```

Comment:
This case is almost identical to `fct2hoogle`. Can we share the code here?

Reply:
Probably going to try and refactor the Hoogle renderer in a future PR.